.editorconfig

```diff
@@ -7,14 +7,14 @@ root = true
 [*]
 charset = utf-8
 indent_style = tab
-indent_size = 2
+indent_size = 3
 trim_trailing_whitespace = true
 insert_final_newline = true
 
 [*.{json,yml,webapp}]
 # npm style
 indent_style = space
-indent_size = 2
+indent_size = 3
```

Comment:
??????

Reply:
Do you see the line above that says "npm style"? Various tools require two-space indentation.